test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlWriterTests.cs

```diff
@@ -2214,6 +2214,323 @@ public void CanWriteAnnotationPathExpression()
 }");
         }
 
+        [Fact]
+        public void CanWriteAnnotationWithoutSpecifiedValue()
+        {
+            // Arrange
+            EdmModel model = new EdmModel();
+            EdmComplexType complex = new EdmComplexType("NS", "Complex");
+            model.AddElement(complex);
+            EdmTerm term1 = new EdmTerm("NS", "MyAnnotationPathTerm", EdmCoreModel.Instance.GetAnnotationPath(false));
+            EdmTerm term2 = new EdmTerm("NS", "MyDefaultStringTerm", EdmCoreModel.Instance.GetString(false), "Property Term", "This is a test");
+            EdmTerm term3 = new EdmTerm("NS", "MyDefaultBoolTerm", EdmCoreModel.Instance.GetBoolean(false), "Property Term", "true");
+
+            model.AddElement(term1);
+            model.AddElement(term2);
+            model.AddElement(term3);
+
+            EdmVocabularyAnnotation annotation = new EdmVocabularyAnnotation(complex, term1, new EdmAnnotationPathExpression("abc/efg"));
+            annotation.SetSerializationLocation(model, EdmVocabularyAnnotationSerializationLocation.Inline);
+            model.SetVocabularyAnnotation(annotation);
+
+            // Succeeds at using term's default value when value not specified (for string/bool)
+            annotation = new EdmVocabularyAnnotation(complex, term2);
+            Assert.True(annotation.UsesDefault);
+            Assert.Equal(annotation.Term.DefaultValue, ((EdmStringConstant)annotation.Value).Value);
+            annotation.SetSerializationLocation(model, EdmVocabularyAnnotationSerializationLocation.Inline);
+            model.SetVocabularyAnnotation(annotation);
+
+            annotation = new EdmVocabularyAnnotation(complex, term3);
+            Assert.True(annotation.UsesDefault);
+            Assert.True(bool.Parse(annotation.Term.DefaultValue));
+            annotation.SetSerializationLocation(model, EdmVocabularyAnnotationSerializationLocation.Inline);

```

Comment:
>        annotation.SetSerializationLocation(model, EdmVocabularyAnnotationSerializationLocation.Inline); [](http://example.com/codeflow?start=4&length=104)

add a variant of the test that doesn't set location to inline. currently I think that test will fail (i.e., it will write the default value in xml).